docker/easy-setup.sh

```diff
@@ -587,6 +617,29 @@ esac
 
 echo
 
+################
+# RESTART MODE #
+################
+
+echo -e "Do you want the containers to restart automatically on startup? [Y/n] "
```

Comment:
Is this taking into account  non interractive setup ?

Reply:
yes, with more precision through the "--restart-mode"